wiki/Beatmaps/en.md

```diff
@@ -3,57 +3,153 @@
+- [Kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
+- [Storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
+- [Custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
+- [Title text](Title_Text), which presents the song's metadata.
+
+## Beatmapsets
+
+*Main article: [Beatmapsets](Beatmapsets)*
+
+On a higher level, beatmaps are grouped into [beatmapsets](Beatmapsets) and then [submitted](/wiki/Glossary#bss) to the [beatmap listing](https://osu.ppy.sh/beatmapsets). Each beatmapset typically covers one song and holds together several beatmaps with different object placement and gameplay settings. In this context, a beatmap is called a [difficulty](/wiki/Difficulties).
+
+While every beatmapset is attributed to [a single person](Beatmapsets/Beatmapset_host), it is often a joint effort: some difficulties may be done by other people working [in collaboration](Beatmap_collaborations) or [separately](Beatmapsets/Guest_difficulty).
+
+Submitted beatmaps may also become [ranked](#ranked) and receive their own leaderboard, for which they have to go through the [ranking procedure](/wiki/Beatmap_ranking_procedure).
+
+## Downloading beatmaps
+
+Beatmaps can be obtained through a few different ways:
+
+### Beatmap listing
+
+Beatmaps from various categories are available from the [beatmap listing](https://osu.ppy.sh/beatmapsets) on the web site. They can be filtered according to certain conditions (for example, beatmaps from [osu!standard](/wiki/Game_mode/osu!) made by a certain mapper) and then downloaded one by one. [osu!supporters](/wiki/osu!supporter) have access to an extended set of filters, such as ability to see beatmaps where they achieved a certain rank.
+
+### osu!direct
+
+osu!direct is the beatmap listing built into the game client, which is used for quick access and provides a set of features similar to the listing on the website. It is available to [osu!supporters](/wiki/osu!supporter) only.
+
+### Beatmap packs
+
+*Main article: [Beatmap Packs](Packs)*
+
+Beatmapsets that were ranked during the same month, or share the same theme (for example, the same artist), are packed for bulk download into archives, which are called beatmap packs. The list of all official packs is available on the [website](https://osu.ppy.sh/beatmaps/packs).
+
+### Unofficial sources
+
+There also exist secondary sources of beatmaps, which are **not checked and not operated by osu!team**. Still, they are available on the forums and appreciated by the community. Such sources include:
+
+- Third-party beatmap mirrors hosted off the platform;
+- Player-made packs and compilations shared via [p2p-networks](https://en.wikipedia.org/wiki/Peer-to-peer "Wikipedia: Peer-to-peer") such as BitTorrent.
```

Comment:
```suggestion
- player-made packs and compilations shared via [p2p-networks](https://en.wikipedia.org/wiki/Peer-to-peer "Wikipedia"), such as BitTorrent.
```

Reply:
I always wanted to ask what's the point of using just a website's title if you can already see the domain name it points to?

Reply:
> I always wanted to ask what's the point of using just a website's title if you can already see the domain name it points to?

Clarification? I think it has something to do with making sure that readers know what website the linked text belongs to in case the text is the name of the article and/or if the domain name changes iirc

Reply:
i guess we decided to add external site names to the title text sometime ago, there should be a convo in the discord somewhere for this i think?

Reply:
> i guess we decided to add external site names to the title text sometime ago, there should be a convo in the discord somewhere for this i think?

just a did quick search. You are correct https://discord.com/channels/188630481301012481/218677502141399041/280008835207397387

Reply:
uhh alright I'll go with that for now